enclave/sgx/attester.c

```diff
@@ -207,7 +224,7 @@ static oe_result_t _get_evidence(
 
 done:
     oe_free(claims);
-    oe_free_report(report);
+    oe_free(report);
```

Comment:
This should remain oe_free_report because report is not allocated in this function.

Reply:
Agreed.